packages/feedback-react/documentation/Example.tsx

```diff
@@ -1,56 +1,48 @@
-    const Component = boolValues?.["Uten smilefjes"] ? SimplifiedFeedback : Feedback;
+const followupQuestions: FeedbackQuestion[] = [
+    {
+        type: "single",
+        label: "Hvordan opplevde du å bestille forsikring på nett?",
+        name: "opplevelse",
+        options: [
+            { label: "Enkelt og greit", value: "enkelt" },
+            { label: "Midt på treet", value: "ok" },
+            { label: "Tungvindt", value: "tungvindt" },
+        ],
+    },
+    {
+        type: "multiple",
+        label: "Hva er viktig for deg?",
+        name: "viktig",
+        options: [
+            { label: "At det går raskt å bestille", value: "hurtighet" },
+            { label: "God informasjon", value: "info" },
+            { label: "At det ser fint ut", value: "utseende" },
+            { label: "At forsikringen er billig", value: "pris" },
+        ],
+    },
+    {
+        type: "text",
+        label: "Er det noe mer du vil legge til?",
+        name: "annet",
+        options: [],
+    },
+];
 
-    return (
-        <>
-            <Component
-                label="Gi oss tilbakemelding!"
-                description="Hvor fornøyd er du med denne siden for å følge saken?"
-                onSubmit={console.info}
-                showTextArea={!boolValues?.["Uten tekst"]}
-                renderCustomSuccess={(props) => (
-                    <div>
-                        <SuccessMessage title="Tilbakemelding sendt!">
-                            {props.value === VERY_UNHAPPY.value && <>Det var trist!</>}
-                            {props.value === UNHAPPY.value && (
-                                <>Vi ser på alle tilbakemeldinger, håper vi kan gjøre deg mer fornøyd en annen gang!</>
-                            )}
-                            {props.value === NEUTRAL.value && (
-                                <>Vi vil gjerne ha fornøyde kunder, så vi skal se på tilbakemeldingen din!</>
-                            )}
-                            {props.value === HAPPY.value && <>Takk skal du ha!</>}
-                            {props.value === VERY_HAPPY.value && (
-                                <>
-                                    Det var stas du var fornøyd, vi prøver hele tiden å bli bedre! Takk for
-                                    tilbakemeldingen!
-                                </>
-                            )}
-                        </SuccessMessage>
-                        {props.message && (
-                            <div className="jkl-layout-spacing--small-top">
-                                <span>Kopi av din melding</span>
-                                <pre>{props.message}</pre>
-                            </div>
-                        )}
-                    </div>
-                )}
-            />
+export const Example = ({ boolValues, choiceValues }: ExampleComponentProps) => {
+    const followup = boolValues?.["Med oppfølgings-spørsmål"]
```

Comment:
```suggestion
    const followUp = boolValues?.["Med oppfølgings-spørsmål"]
```
Her er det verb :smile: